### PR TITLE
Support rustc 1.0.0-nightly (d754722a0 2015-03-31)

### PR DESCRIPTION
**Pull request by Aigeruth**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -36,7 +36,7 @@ Default options are `build` and `test`
 ");
 
 #[derive(Copy)]
-struct Config {
+pub struct Config {
```
Comment (neuschaefer) [inline src/main.rs]: Why is this necessary?

File: Cargo.toml [hunk 2]
```diff
@@ -25,6 +25,9 @@ keywords = [
 rustc-serialize = "*"
 docopt = "0.6"
 docopt_macros = "0.6"
-notify = "^1.1.3"
 regex = "*"
 time = "*"
+
+[dependencies.notify]
```
Comment (passcod) [inline Cargo.toml]: Don't do that. I'll crate up notify soon, if you want me to do that sooner remind me about it, but don't make this Cargo.toml depend on non- crates.io things, please.

File: src/main.rs (hunk 1 above)
Comment (passcod) [inline src/main.rs]: Actually I just tried this and the compiler complains about a `private type in exported type signature` if we don't specify `pub`.
